unit-test/background/pixel.es6.js

```diff
@@ -0,0 +1,60 @@
+
+const pixel = require('../../shared/js/background/pixel.es6')
+const url = 'https://improving.duckduckgo.com/t/'
+const getURLTestCases = [
+    {
+        'pixelName': 'ep',
+        'result': url + 'ep'
+    },
+    {
+        'pixelName': '',
+        'result': 'undefined'
+    },
+    {
+        'pixelName': 'undefined',
+        'result': 'undefined'
+    }
+]
+const concatParamsTestCases = [
+    {
+        'params': [],
+        'partialResult': ''
+    },
+    {
+        'params': [
+            'param1'
+        ],
+        'partialResult': '_param1'
+    },
+    {
+        'params': [
+            'param1',
+            'param2'
+        ],
+        'partialResult': '_param1_param2'
```

Comment:
Can we add more test cases here? I'd like to see test cases for the two things from my comment above, as well as a test case for multiple object params like:

```js
[{ foo: 'bar' }, { bar: 'baz' }]
```

Reply:
Note, if you add a `Math.random()` as per my earlier comment above you'll probably need to add something like:

```js
spyOn(Math, 'random').and.returnValue(1)
```

so you don't need to worry about having to match the random part with a regex or something.